mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -28,6 +28,8 @@ class ExperimentPage extends Component {
         paramKeyFilterString: urlState.params === undefined ? "" : urlState.params,
         metricKeyFilterString: urlState.metrics === undefined ? "" : urlState.metrics,
         searchInput: urlState.search === undefined ? "" : urlState.search,
+        orderByKey: urlState.orderByKey === undefined ? null : urlState.orderByKey,
```

Comment:
I think we need to cast these from strings to bools, else they result in this React warning when passed through as props to ExperimentView -> ExperimentRunsTableCompactView:
```
index.js:2178 Warning: Failed prop type: Invalid prop `orderByAsc` of type `string` supplied to `ExperimentRunsTableCompactView`, expected `boolean`.
    in ExperimentRunsTableCompactView (created by Connect(ExperimentRunsTableCompactView))
    in Connect(ExperimentRunsTableCompactView) (at ExperimentView.js:468)
    in div (at ExperimentView.js:311)
    in div (at ExperimentView.js:289)
    in ExperimentView (created by Connect(ExperimentView))
    in Connect(ExperimentView) (created by Route)
    in Route (created by withRouter(Connect(ExperimentView)))
    in withRouter(Connect(ExperimentView)) (at ExperimentPage.js:167)
    in RequestStateWrapper (created by Connect(RequestStateWrapper))
    in Connect(RequestStateWrapper) (at ExperimentPage.js:144)
    in div (at ExperimentPage.js:143)
    in ExperimentPage (created by Connect(ExperimentPage))
    in Connect(ExperimentPage) (created by Route)
    in Route (created by withRouter(Connect(ExperimentPage)))
    in withRouter(Connect(ExperimentPage)) (at HomeView.js:58)
    in div (at HomeView.js:56)
    in div (at HomeView.js:47)
    in HomeView (created by Connect(HomeView))
    in Connect(HomeView) (at HomePage.js:26)
    in RequestStateWrapper (created by Connect(RequestStateWrapper))
    in Connect(RequestStateWrapper) (at HomePage.js:25)
    in HomePage (created by Connect(HomePage))
    in Connect(HomePage) (created by Route)
    in Route (at App.js:55)
    in Switch (at App.js:49)
    in AppErrorBoundary (at App.js:48)
    in div (at App.js:24)
    in Router (created by HashRouter)
    in HashRouter (at App.js:23)
    in App (created by Connect(App))
    in Connect(App) (at index.js:14)
    in Provider (at index.js:13)
```